### ISSUE TITLE
questions about scaffold 

### ISSUE THREAD

**Issue by seuliang** [michael-brade/LaTeX.js#84]
`rsync` can't be use'd at windows:
https://github.com/michael-brade/LaTeX.js/blob/ad1f84fb411a71aed34ca28ba66a5183a9dd1621/package.json#L37

what does this part do？
https://github.com/michael-brade/LaTeX.js/blob/ad1f84fb411a71aed34ca28ba66a5183a9dd1621/rollup.config.js#L19

and how did you deal with this part？
https://github.com/michael-brade/LaTeX.js/blob/ad1f84fb411a71aed34ca28ba66a5183a9dd1621/src/latex.ltx.ls#L1189

**Comment by michael-brade** [comment]
Good questions! The first one is my doing.... I don't have Windows, so I didn't care much about it. I guess `rsync` could be replaced by some rollup code. Maybe it could be conditional? On Linux, use `rsync`, on Windows use something else that's appropriate.

Regarding the other two questions, maybe @fgborges can answer those, he implemented that. I haven't taken the time to investigate yet.

**Comment by seuliang** [comment]
use nodejs package to do that will be better just like `rimraf` and `mkdirp `

**Comment by michael-brade** [comment]
Yeah... but there is no such thing as an rsync package, only `rsync` wrappers, which means you need to have cygwin installed. Maybe I'll just use the rollup plugin... it's only 8MB that has to be copied, so that should be fast enough.

**Comment by seuliang** [comment]
yes,  using copy is enough.

about second and third question: can we just import all documentclasses into latex.ltx.ls, rather than import it dynamically, and choose one type according to input？
